examples/example_apple_opengl2/Makefile

```
@@ -0,0 +1,21 @@
+# Makefile for example_apple_metal, for macOS only (**not iOS**)
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)